setup.py

```diff
@@ -2,12 +2,12 @@
 
 setup(
     name='autograd',
-    version='1.1.9',
+    version='1.1.10',
     description='Efficiently computes derivatives of numpy code.',
     author='Dougal Maclaurin and David Duvenaud and Matthew Johnson',
     author_email="maclaurin@physics.harvard.edu, duvenaud@cs.toronto.edu, mattjj@csail.mit.edu",
     packages=['autograd', 'autograd.numpy', 'autograd.scipy', 'autograd.scipy.stats'],
-    install_requires=['numpy>=1.10', 'scipy>=0.17', 'future>=0.15.2'],
+    install_requires=['numpy>=1.12.1', 'scipy>=0.17', 'future>=0.15.2'],
```

Comment:
I had only tested it on 1.12.1, but I just downgraded and made sure it indeed works on 1.12!